PSICover/PSICover.cs

```diff
@@ -160,21 +162,70 @@ class Analyzer {
                blocks.RemoveAt (i - 1);
          blocks.Reverse ();
 
+         int hitscount = 0;
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
          foreach (var block in blocks) {
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
-            code[block.ELine] = code[block.ELine].Insert (block.ECol, "</span>");
-            code[block.SLine] = code[block.SLine].Insert (block.SCol, tag);
+            if (hit) hitscount++;
+            string endtag = hit ? $"<span class=\"tooltiptext\">{hits[block.Id]}hits</span></div>" : "</div>";
+            string starttag = $"<div class=\"{(hit ? "hit" : "unhit")}\">";
+            int multilines = block.ELine - block.SLine;
+            if (multilines > 0) {

```

Comment:
If condition is not required

Reply:
updated, removed the multilines

![image](https://github.com/SasiTMM/PSI/assets/125174306/46384a39-0cf6-4ceb-a083-b379cdd0d574)